en/docs/guides/adaptive-auth/configure-adaptive-auth.md

```diff
@@ -78,4 +69,9 @@ function someCommonFunction(context) {
 
 - To learn about the guidelines on writing custom functions for adaptive authentication, see [Writing Custom Functions for Adaptive Authentication](insertlink).
 
-----
\ No newline at end of file
+
+!!! info "Related Topics"
+    - [Concept: Adaptive-Authentication](TODO:insert-link-to-concept)
+    - [Guide: Ensure Assurance with Authentication Context Class Reference (ACR)  and Authentication Method Reference (AMR)](../../adaptive-auth/work-with-acr-amr)
```

Comment:
can we shorten this to "Ensure Assurance with ACR/AMR" like we have titled it in the mkdocs.yml ?